infra/dockerfile.spawn_data_pipeline

```diff
@@ -0,0 +1,11 @@
+#syntax=docker/dockerfile:experimental
+FROM python:3.7.3-slim
+
+# Setup dependencies in a cachable step
+RUN pip install --disable-pip-version-check --no-cache-dir taskcluster==7.0.1 json-e==3.0.0 requests==2.21.0 pyyaml==5.1
+
+ADD taskcluster-hook-pipeline-start.json /code
```

Comment:
This should be data-pipeline.yml, which should be passed to spawn_data_pipeline.py